src/WorkBC.Web/WorkBC.Web.csproj

```diff
@@ -66,6 +66,9 @@
     <Content Remove="ClientApp\projects\jb-search\src\app\services\**" />
     <Content Remove="ClientApp\src\**" />
     <None Include="$(SpaRoot)**" Exclude="$(SpaRoot)node_modules\**" />
+    <Compile Remove="wwwroot\**" />
+    <Content Remove="wwwroot\**" />
+    <None Remove="wwwroot\**" />
```

Comment:
@JonTaylor1971 any thoughts on why this file shows as changed?  I'm considering rolling back the changes.